drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/base_loader.rb

```diff
@@ -161,9 +161,9 @@ def inspect
       self.class.name.to_s
     end
 
-    def log_skipped_row(row, field:)
+    def log_skipped_row(row, field:, prefix: nil)
       value = row_value(row, field: field)
-      log_info "#{row.context} could not resolve \"#{field}\":\"#{value}\""
+      log_info "#{row.context} #{prefix || ''} could not resolve \"#{field}\":\"#{value}\""
```

Comment:
this is fine. Note: in ruby `nil.to_s` is an empty string so you can just say "#{row.context} #{prefix}"

Reply:
oh oops looks like this is the wrong base_loader

Reply:
good catch!